src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -249,6 +249,7 @@ class WorkspaceMembersPage extends React.Component {
         const data = _.chain(policyEmployeeList)
             .map(email => this.props.personalDetails[email])
             .filter()
+            .unique()
```

Comment:
NAB: Just to understand why are we looking for unique emails, are we not handling while one tries to invite someone to the workspace?

Reply:
I think we can remove this as displaying duplicate logins would indicate an issue elsewhere that should probably be fixed.